src/meta/src/hummock/compaction/mod.rs

```diff
@@ -174,6 +174,12 @@ impl CompactStatus {
     }
 
     /// Declares a task as either succeeded, failed or canceled.
+    pub fn report_compact_task_pb(&mut self, compact_task: &PbCompactTask) {
```

Comment:
When will it be used? When we use it, can be first convert it to `CompactTask` and then call the `report_compact_task`?